### PR TITLE
Rename trigger to triggering_action

### PR DESCRIPTION
**Pull request by akofink**
The trigger method conflicts with that of WithSubTasks

### CODE REVIEW

File: lib/dynflow/action.rb [hunk 1]
```diff
@@ -165,11 +165,11 @@ def caller_action
     def set_plan_context(execution_plan, trigger, from_subscription)
       phase! Plan
       @execution_plan    = Type! execution_plan, ExecutionPlan
-      @trigger           = Type! trigger, Action, NilClass
+      @trigger           = Type! triggering_action, Action, NilClass
```
Comment (iNecas) [inline lib/dynflow/action.rb]: the instanca variable shoud be renamed as well

**Comment by iNecas** [comment]
It seems the tests need to be updated as well

**Comment by akofink** [comment]
@iNecas Could you please take a look at the test failures? The majority of the environments are passing, and it's passing locally for me. No single database or Ruby version is failing, so I'm a bit confused.

**Comment by adamruzicka** [comment]
@akofink sadly, when running tests on travis we get those kind of errors from time to time. Restarted the failed ones and they're green now.

**Comment by akofink** [comment]
Good to know! Thanks!

**Comment by iNecas** [comment]
Thanks @akofink, do you have changes in Katello ready so that I can release a new version that you could depend on?

**Comment by akofink** [comment]
I will mention you on the PR once submitted.

**Comment by jlsherrill** [comment]
@iNecas when we built this new version of dynflow, can you make sure it gets built for 1.15?

**Comment by jlsherrill** [comment]
@iNecas actually i went ahead and opened a pr for that

**Pull request merged**